packages/victory-core/src/victory-container/victory-container.js

```diff
@@ -145,18 +151,19 @@ export default class VictoryContainer extends React.Component {
   }
 
   render() {
-    const { width, height, responsive, events, title, desc, tabIndex } = this.props;
+    const { role, width, height, responsive, events, title, desc, tabIndex, ariaLabel } = this.props;
     const style = responsive
       ? this.props.style
       : Helpers.omit(this.props.style, ["height", "width"]);
     const svgProps = assign(
       {
         width,
         height,
-        tabIndex,
-        role: "img",
+        role: role ? role : "img",
```

Comment:
Might make sense to move "img" to `defaultProps` above since this component is using PropTypes.

Reply:
nice point, thanks!